Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -287,6 +287,29 @@ bool GameListItem::ReadPNGBanner(const std::string& path)
   return true;
 }
 
+const std::string GameListItem::GetFileType() const
+{
+  switch (GetBlobType())
+  {
+  case DiscIO::BlobType::PLAIN:
+    return wxTRANSLATE("Plain");
```

Comment:
[wxTRANSLATE](http://docs.wxwidgets.org/3.1/group__group__funcmacro__string.html#gab8862de0f74049708bfea75ae9e14c45) doesn't actually load the translated string, it only marks them for translation. The macro to mark and load translated strings is the `_()` macro.

e.g. `return _("Plain");`.

Also note that this returns a `const wxString&`, not a `std::string`.

cc @JosJuice for string encoding related stuff (if it matters)